modules/tinymce/src/core/test/ts/browser/init/ShadowDomEditorTest.ts

```diff
@@ -18,12 +19,16 @@ const skinSheetsTest = (extraSettings: Record<string, any>) => (success, failure
   TinyLoader.setupInShadowRoot((editor: Editor, shadowRoot: SugarElement<ShadowRoot>, onSuccess, onFailure) => {
     Pipeline.async({}, [
       Step.sync(() => {
-        // TODO TINY-6144: Test that there are no skin stylesheets in the head. We will need to clean up existing stylesheets first, which may require a StyleSheetLoader.removeAll() function
         Assert.eq(
           'There should be a skin stylesheet in the ShadowRoot',
           true,
           Arr.exists(shadowRoot.dom.styleSheets, isSkin)
         );
+        Assert.eq(
+          'There should only be a shadowdom specific skin stylesheet in the document',
```

Comment:
I believe this test will pass if at least one stylesheet is a shadowdom stylesheet, but doesn't actually assert that it is the only skin?

Reply:
Forgot to update the comment/label here. I originally tried to use `Arr.forall`, but that failed because of the bedrock stylesheets. I'll remove `only` from the label.

Reply:
Fixed